README.md

```diff
@@ -69,5 +70,70 @@ Usage:
+* Any modification should be done to the testing branch, and wait at least for one approval for one member of the Apps group. So that we can ensure that there’s nothing in opposition to those criteria. Nor any changes that would harm servers.
+* The package should comply with the [requirement of the level 8](https://github.com/YunoHost/doc/blob/master/packaging_apps_levels.md#level-8).
+
+You can find the validation form used by Apps group [here](https://github.com/YunoHost/apps/blob/master/hq_validation_template.md).
+
+If the app is already tag as High Quality and one of those criteria isn't respected anymore. After a warning, the tag will be removed until the criteria are again validated.
+
+To make an app a High Quality app, technically, you have to add the tag ```"high_quality": true```.
+```json
+    "wallabag": {
+        "branch": "master",
+        "high_quality": true,
+        "revision": HEAD,
+        "url": "https://github.com/abeudin/wallabag_ynh.git",
+        "state": "working"
+    }
+```
+
+### How to make my app a Featured app ?
+
+A Featured app highlighted in the app list and shown before any others.  
+To become a Featured app, a package have to follow the following rules:
+
+* The app should already be a High Quality app.
+* The upstream app should be accessible and well made.
+* The app should be interesting and demanded by the community.
+* The app should fit the spirit of YunoHost.
+
+**Please note that the exact process to decide which app are going to be Featured, and for how many time, isn't yet defined...**
+
+To make an app a Featured app, technically, you have to add the tag ```"featured": true```.
+```json
+    "wallabag": {
+        "branch": "master",
+        "high_quality": true,
+        "featured": true,
+        "revision": HEAD,
+        "url": "https://github.com/abeudin/wallabag_ynh.git",
+        "state": "working"
+    }
+```
+
+### What to do if I can't maintain my app anymore ?
+
+If you don't have time anymore to maintain an app, you can update its status to inform users and packagers that you will not maintain it anymore.  
+In order to do so, use the tag `"maintained":`.  
+This tag can have 5 different values:
+- `"maintained": true` That's the default value if the tag isn't present for your app. That simply means that this app is maintained.
+- `"maintained": "request_help"` Use that value to inform other packagers that you need help to maintain this app. You'll then be more than one maintainer for this apps.
+- `"maintained": "request_adoption"` Use that value to inform other packagers, as well as users, that you're going to give up that app. So that you would like another maintainer to take care of it.
+- `"maintained": false` or `"maintained": "orphaned"` This value means that this app is no longer maintained... That means also that a packager can declare himself as its new maintainer.  
```

Comment:
```suggestion
- `"maintained": false` or `"maintained": "orphaned"` This value means that this app is no longer maintained... That means also that a packager can declare himself/herself as its new maintainer.  
```

Reply:
...
Hope we're not going to start to do that every where in English as well...